Website/src/rendering/.babelrc

```diff
@@ -0,0 +1 @@
+{"presets": ["next/babel"]}
```

Comment:
What is this new file allowing us to do?

Reply:
This is a workaround to an issue that came up as part of migrating to to nextjs v12 and is documented [here](https://nextjs.org/docs/messages/failed-loading-swc#possible-ways-to-fix-it). I will plan on adding some comments to this file